x-pack/plugins/enterprise_search/server/routes/enterprise_search/engines_search.test.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockDependencies, mockRequestHandler, MockRouter } from '../../__mocks__';
+
+import { registerEnginesSearchRoutes } from './engines_search';
+
+describe('engines search routes', () => {
```

Comment:
It seems like we're picking up the Engines CRUD API routes here and not search. This happens regardless of whether it's split up into its own file or not. @TattdCodeMonkey Any ideas what might be going on here?